components/HowItWorks/HowItWorks.tsx

```diff
@@ -124,22 +111,20 @@ export function HowItWorks() {
             <>
               You have voted in{" "}
               <Strong>
-                {countCorrectVotes === undefined ||
-                countWrongVotes === undefined ? (
-                  <LoadingSkeleton width={50} />
-                ) : (
-                  formatNumberForDisplay(getTotalVotes(), { decimals: 0 })
-                )}
+                <Loader
+                  isLoading={isAnyUndefined(countCorrectVotes, countWrongVotes)}
```

Comment:
this usage is only temporary — once this is merged I can go ahead with refactoring these things into hooks, and those hooks will be able to provide these `isLoading` values in a much cleaner way.